blockchain/README.md

```diff
@@ -57,7 +57,7 @@ is by no means exhaustive:
 
 ## Examples
 
-* [ProcessBlock Example](https://godoc.org/github.com/decred/dcrd/blockchain#example-BlockChain-ProcessBlock)
+* [ProcessBlock Example](https://pkg.go.dev/github.com/decred/dcrd/blockchain#example-BlockChain-ProcessBlock)
```

Comment:
Same goes for the examples, and it looks like the link format has changed too.

This should be:

https://pkg.go.dev/github.com/decred/dcrd/blockchain/v3#example-BlockChain.ProcessBlock